hw/top_earlgrey/data/ip/chip_uart_testplan.hjson

```diff
@@ -0,0 +1,252 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_uart
+  testpoints: [
+    // Basic smoke tests:
+    {
+      name: chip_sw_uart_tx_rx_smoke
+      desc: '''Verify the transfer of data from UART
+
+               - Powerup OT and enable UART IO.
+               - Calculate CONTROL.NCO using programmers model and update the register
+               Receive sequence:
+                 - Set FIFO_CTRL.RXILVL to 1 so that one character can be received
+                 - Enable RX
+                 - Transfer a single packet from testbench to UART IO
+                 - Check if there is any interrupt and if the source is from UART
+                 - Check if FIFO_STATUS.RXLVL is 1
+                 - Clear the rx_watermark interrupt
+                 - Repeat the sequence with 10 packets in a single transaction
+
+               Transmit sequence:
+                 - Set FIFO_CTRL.TXILVL to 1 so that one character can be transmitted
+                 - Write a random data to WDATA register
+                 - Enable TX
+                 - Monitor UART IO for transmission of data packet
+                 - Check if the packet framing is correct
+                 - Check if there is any interrupt and if the source is from UART
+                 - Check if there is tx_empty interrupt after transmission of one packet
+                 - Clear the rx_empty interrupt
+                 - Repeat the sequence with 10 packets in a single transaction
+            '''
+      stage: V1
+      si_stage: SV1
+      lc_states: ["TEST_UNLOCKED", "PROD"]
+      features: ["UART.RX_EN", "UART.TX_EN", "UART.RX_WATERMARK_INTERRUPT", "UART.TX_WATERMARK_INTERRUPT",
+                 "UART.BAUD_RATE_CONTROL"]
+      tests: ["chip_sw_uart_tx_rx_smoke"]
+    },
+    {
+      name: chip_sw_uart_tx_rx_parity_smoke
+      desc: '''Verify the transfer of data from UART with parity enabled
+
+               - Powerup OT and enable UART IO.
```

Comment:
This is (sensibly!) very similar to the description for `chip_sw_uart_tx_rx_smoke`. Would it make sense to shorten this one dramatically by just listing the differences? It could be something like "The same as chip_sw_uart_tx_rx_smoke, except that ..." ?

Reply:
Updated description